Formula/folly.rb

```diff
@@ -51,7 +60,8 @@ class Folly < Formula
   def install
     ENV.llvm_clang if OS.mac? && (DevelopmentTools.clang_build_version <= 1100)
 
-    args = std_cmake_args + %w[
+    args = std_cmake_args + %W[
+      -DCMAKE_LIBRARY_ARCHITECTURE=#{Hardware::CPU.arch}
```

Comment:
What does this do, exactly? The [documentation](https://cmake.org/cmake/help/latest/variable/CMAKE_LIBRARY_ARCHITECTURE.html) doesn't say too much.

Reply:
I don't think it does anything meaningful via CMake, but not entirely sure.

It gets set on OS like Debian that use `/usr/lib/x86_64-linux-gnu` directories.

The problem is `folly` automatically assumes x86_64 if `CMAKE_LIBRARY_ARCHITECTURE` is not set.